PowerPointLabs/PowerPointLabs/Utils/SlideUtil.cs

```diff
@@ -119,12 +121,13 @@ public static Design GetDesign(string designName)
         public static void CopyToDesign(string designName, PowerPointSlide refSlide)
         {
             Design design = GetDesign(designName);
-            if (design == null)
+            if (design != null)
             {
-                design = CreateDesign(designName);
+                design.Delete();
```

Comment:
see if can add a try-catch, in case design == null and you get a NullReferenceException

@alexfjw Help to test on 2016

Reply:
Hmm I don't think there will be a NullReferenceException because I check if design is not equals to null. However, ok I'll add a Try-Catch anyway because I'm not sure under what circumstances the design cannot be deleted. (There isn't an API documentation)